roles/analysis/README.md

```diff
@@ -1,27 +1,65 @@
 # Analysis
 
-The `analysis` role is used to create the Leapp pre-upgrade report on the target hosts. It also saves a copy of the current ansible facts under the `/etc/ansible/facts.d` directory for validation after upgrade.
+The `analysis` role is used to create the Leapp pre-upgrade report on the target hosts.
 
-The role is considered minimally invasive and hopefully will fly under the radar of your enterprise change management policy. That said, it does install the RHEL rpm packages that provide the Leapp framework if they are not already present. While application impact is very low, it may require a change ticket depending on how pedantic your policies are.
+This is used in IPU planning to identify unhandled cases, that would result in process failure.
+
+This role also saves a copy of the current ansible facts under the `/etc/ansible/facts.d` directory for validation after upgrade.
 
 ## Requirements
 
-For non-CDN environments including Satellite, package metadata files needs to be provided. These files can be put on the host under the `/etc/leapp/files` directory before running the role. Alternatively, the `leapp_metadata_url` can be defined to tell the role where it can access the metadata tarball. For more information, refer to the knowledge article [Leapp utility metadata in-place upgrades of RHEL for disconnected upgrades (including Satellite)](https://access.redhat.com/articles/3664871).
+Metadata is required for leap preupgrade analysis to run.  When a system uses CDN, this is provided automatically.  For systems not on CDN, package metadata files needs to be provided by URL in leapp_metadata_url. Once obtained during IPU, file is unpacked on IPU host under `/etc/leapp/files` directory.
+
+In an upcomoing release, the metadata will be included as part of the RPM install of leapp RPMs.
+
+For more information, refer to the knowledge article [Leapp utility metadata in-place upgrades of RHEL for disconnected upgrades (including Satellite)](https://access.redhat.com/articles/3664871).
+
+## Commonly modified variables
+
+| Name                  | Type | Default value           | Description                                     |
+|-----------------------|------|-------------------------|-------------------------------------------------|
+| leapp_upgrade_type    | String  | satellite | satellite, cdn or rhui |
+| leapp_preupg_opt | String | --no-rhsm when leapp_upgrade_type is connected | Upstream repository usage - whether to use RHSM |
```

Comment:
Previous description was better. Explanation of the default value is confusing. If we really want to provide a way for users to append extra options to the leapp command, we should use a different variable that won't interfere with our default logic. This is an optional variable, so it should go in your new optional section. 

Move to optional section and change to this...
```suggestion
| leapp_preupg_opt | String |  | Optional string to define command line options to be passed to the `leapp` command when running the pre-upgrade. |
```